core/src/main/antlr4/com/databricks/labs/remorph/parsers/lib/basesnowflake.g4

```diff
@@ -0,0 +1,204 @@
+// =================================================================================
+// Please reformat the grammr file before a change commit. See remorph/core/README.md
+// For formatting, see: https://github.com/mike-lischke/antlr-format/blob/main/doc/formatting.md
+
+// $antlr-format alignTrailingComments true
+// $antlr-format columnLimit 150
+// $antlr-format maxEmptyLinesToKeep 1
+// $antlr-format reflowComments false
+// $antlr-format useTab false
+// $antlr-format allowShortRulesOnASingleLine true
+// $antlr-format allowShortBlocksOnASingleLine true
+// $antlr-format minEmptyLines 0
+// $antlr-format alignSemicolons ownLine
+// $antlr-format alignColons trailing
+// $antlr-format singleLineOverrulesHangingColon true
+// $antlr-format alignLexerCommands true
+// $antlr-format alignLabels true
+// $antlr-format alignTrailers true
+// =================================================================================
+lexer grammar basesnowflake;
```

Comment:
move this into `snowflake` directory. same for `basetsql`.

Reply:
I went one better and removed the need for it by merging more of the lexers into a single unified lexer